test-project/Assets/Generated/Source/improbable/Coordinates.cs

```diff
@@ -0,0 +1,56 @@
+// ===========
```

Comment:
Was this intended to be committed?

Reply:
I must not have ignored the std lib generated stuff - not